### ISSUE TITLE
create_src_directory.py missing

### ISSUE THREAD

**Issue by nicoonoclaste** [compiler-research/cppyy-backend#103]
`setup.py` attempts to run a script named `create_src_directory.py`, which does not exist in the repository.

**Comment by vgvassilev** [comment]
@nbraud, are you sure you want to use our work-in-progress fork and not the main repository?

**Comment by vgvassilev** [comment]
@nbraud, CppInterOp with the clang-repl backend should be fine to package. Why did you need patches of llvm for the clang-repl backend?

I am not sure if we want to package our fork of cppyy because it will cause confusion. @wlav, what do you think?

**Comment by wlav** [comment]
Your fork doesn't work, does it? Once it passes all tests, it should be fine, if properly versioned.

**Comment by vgvassilev** [comment]
> Your fork doesn't work, does it? Once it passes all tests, it should be fine, if properly versioned.

Yes, that fork works partially and that is why it is not merged upstream. Perhaps it is not the right thing to ship a package.

**Comment by vgvassilev** [comment]
> > Why did you need patches of llvm for the clang-repl backend?
> 
> The build instructions for CppInterOp mentioned needing to apply patches from `patches/llvm`. Is that not the case when using the `clang-repl` backend and/or for recent-enough LLVM versions? I have to admit it would make things easier, packaging-wise.

If you build it against llvm/clang-18 it should not need any patches.
